src/Tools/AI Test Toolkit/src/AITTestContextImpl.Codeunit.al

```diff
@@ -146,6 +149,33 @@ codeunit 149043 "AIT Test Context Impl."
         SetSuiteTestOutput(CurrentTestOutputJson.ToText());
     end;
 
+    /// <summary>
+    /// Sets the accuracy of the test.
+    /// </summary>
+    /// <param name="AccuracyPct">The accuracy as a decimal between 0 and 1.</param>
+    procedure SetAccuracy(AccuracyPct: Decimal)
+    begin
+        if (AccuracyPct < 0) or (AccuracyPct > 1.0) then
+            Error(AccuracyErr);
+
+        AccuracySetManually := true;
+        Accuracy := AccuracyPct;
+    end;
+
+    /// <summary>
+    /// Gets the accuracy of the test.
+    /// </summary>
+    /// <returns>True if the accuracy was set, false otherwise.</returns>
```

Comment:
missing parameter description for AccuracyPct

Reply:
It is not clear that this function returns accuracy only when it is being set manually.